test/MFEMHesthavenComparison/TestMfemHesthavenFunctions.h

```diff
@@ -4,20 +4,19 @@
 #include "mfem.hpp"
 #include <fstream>
 #include <iostream>
-#include <../../maxwell/src/maxwell/Types.h>
-#include <../../maxwell/src/maxwell/BilinearIntegrators.h>
+#include "maxwell/Types.h"
+#include "maxwell/BilinearIntegrators.h"
 #include "gtest/gtest.h"
 
 #include <Eigen/Dense>
 #include <maxwell/Model.h>
 
 
 std::unique_ptr<FiniteElementSpace> buildFiniteElementSpace(const int order);
-Eigen::MatrixXd	buildNormalPECFluxOperator1D(std::unique_ptr<FiniteElementSpace>& fes, std::vector<maxwell::Direction> dirVec);
 Eigen::MatrixXd buildExpectedAverageDenseMatrix1D(const int order,const int elements);
 Eigen::MatrixXd buildExpectedJumpDenseMatrix1D(const int order, const int elements);
-Eigen::MatrixXd buildEigenDGTrace1D(std::unique_ptr<FiniteElementSpace>& fes, std::pair<double, double> ab);
-Eigen::MatrixXd buildEigenMaxwellDGTrace1D(std::unique_ptr<FiniteElementSpace>& fes, std::vector<maxwell::Direction> dir, const double beta);
+Eigen::MatrixXd buildEigenDGTrace1D(FiniteElementSpace* fes, std::pair<double, double> ab);
+Eigen::MatrixXd buildEigenMaxwellDGTrace1D(FiniteElementSpace* fes, std::vector<maxwell::Direction> dir, const double beta);
```

Comment:
const FiniteElementSpace* ?

Reply:
BilinearForm doesn't accept const FiniteElementSpace* as input argument, it could be a simple FiniteElementSpace or it could be left as is.